apps/user/migrations/0004_auto_20171129_0851.py

```diff
@@ -15,4 +15,5 @@ class Migration(migrations.Migration):
     operations = [
         CreateExtension('pg_trgm'),
         CreateExtension('postgis'),
+        CreateExtension('unaccent'),
```

Comment:
This will not work as this migration is already run in all of the existing systems. You need to create a new one.
I recommend looking at the Django documentation https://docs.djangoproject.com/en/4.2/topics/migrations/ for full context.

Reply:
Let's remove this.

Reply:
@thenav56 

> Let's remove this.

This is removed .